src/Faker/Core/Medical.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Faker\Core;
+
+use Faker\Extension;
+
+/**
+ * @experimental This class is experimental and does not fall under our BC promise
+ */
+final class Medical implements Extension\MedicalExtension
```

Comment:
Perhaps a `Blood` extension would make more sense?

All of the methods in this class share the `blood` prefix.

Reply:
Yeah, that would make more sense.

Reply:
Renamed and reword to Blood instead